machine/get-started.md

```diff
@@ -28,23 +28,23 @@ of these, you get `docker-machine`.
 
 The new solutions come with their own native virtualization solutions rather
 than using Oracle VirtualBox, so there are new considerations to keep in mind
-when using Machine to create local VMs. Docker for Mac allows for the creation
-of additional `docker-machine` based `virtualbox` machines without issue, but in
-the case of Docker for Windows, you will need to use the `hyperv` driver
-instead.
+when using Machine to create local VMs.
 
+- Docker for Mac: You can create additional `docker-machine` machines using the
+  `virtualbox` driver.
+- Docker for Windows: You can create additional `docker-machine` machines, but you
```

Comment:
I would simplify to:

Docker for Windows: You can create additional `docker-machine` machines using the `hyperv` driver.